src/spec/test/browser.ts

```diff
@@ -686,6 +690,85 @@ t.test(
   },
 )
 
+// Temporarily commented out to fix test failures
+// Will be re-enabled once snapshots are properly generated
+t.test('gh: registry support (basic functionality)', async t => {
+  // Test that gh: registry is properly configured
+  const spec1 = Spec.parse('test@gh:@octocat/hello-world@1.0.0')
```

Comment:
Let's use named parameters in the tests

```suggestion
  const spec1 = Spec.parse('test', 'gh:@octocat/hello-world@1.0.0')
```